pkg/gds/service.go

```diff
@@ -16,9 +16,6 @@ import (
 )
 
 func init() {
-	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
-	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
```

Comment:
This was some stray code that gets overridden in lines 21 and 26 so it needed to be removed.